.github/workflows/private-elixir-library.yml

```diff
@@ -198,15 +198,6 @@ jobs:
         with:
           name: ${{ steps.app-name.outputs.app-name }}-docs-${{ steps.app-version.outputs.app-version }}.tar.gz
           path: docs.tar.gz
-      - name: Creating new Github release
-        id: create-release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: v${{ steps.app-version.outputs.app-version }}
-          release_name: Release ${{ steps.app-name.outputs.app-name }} ${{ steps.app-version.outputs.app-version }}
-          body: Autogenerated from tag push
       - name: Uploading hex package as release artifact
```

Comment:
Not quite. These release artifact steps have to happen after the release is created.

Reply:
Right, I'll revert, no worries